### PR TITLE
Use Dependency Injection for User Feedback on ASP.NET Core

### PR DESCRIPTION
**Pull request by kanadaj**
The documentation should avoid using hard-coded DSNs in places where configuration settings are available as a single source of truth.

### CODE REVIEW

**Comment by untitaker** [comment]
`___PUBLIC_DSN___` is a placeholder that is dynamically replaced in the docs with the real DSN. I would keep that placeholder somewhere.

**Comment by kanadaj** [comment]
@untitaker Ah I see. I was a bit on the fence about whether I should keep a note about setting up without DI. Will amend in a moment

File: src/collections/_documentation/enriching-error-data/user-feedback-example/aspnetcore.md [hunk 1]
```diff
@@ -5,12 +5,19 @@ With ASP.NET Core MVC, the `Error.cshtml` razor page:
...
 {
     <script>
-        Sentry.init({ dsn: '___PUBLIC_DSN___' });
+        Sentry.init({ dsn: '@(SentryOptions.Value.Dsn)' });
         Sentry.showReportDialog({ eventId: '@SentrySdk.LastEventId' });
     </script>
 }
 ```
+
+If you configured Sentry without Dependency Injection (e.g. without `UseSentry()`) you can also configure the SDK using a hard-coded DSN in the template:
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/user-feedback-example/aspnetcore.md]:
```suggestion
If you configured Sentry without the Dependency Injection (for example, without `UseSentry()`) you can also configure the SDK using a hard-coded DSN in the template:
```

**Comment by kanadaj** [comment]
@MimiDumpling thank you, I actually wasn't sure about consistency since I'm not a regular contributor for for the docs :) Added the suggestions as a commit!

**Comment by bruno-garcia** [comment]
Thanks @kanadaj !

**Pull request merged**